### PR TITLE
ON-28844 # Removed `isInfoPage` prop

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: typescript/forms.d.ts [hunk 1]
```diff
@@ -800,8 +800,6 @@ export type Form = {
   publishStartDate?: string
   /** The date and time (in ISO format) a form becomes unavailable. */
   publishEndDate?: string
-  /** Whether or not the Form is an Info Page. */
```
Comment (kizaonline) [inline typescript/forms.d.ts:803]: Need to also remove `InfoPageElement`

Comment (mymattcarroll) [reply typescript/forms.d.ts:803]: When we remove the `InfoPageElement`, can we please change the `FormFormElement['type']` to  `'form' | 'infoPage'`.

Comment (Zaxist) [reply typescript/forms.d.ts:803]: Why do we need that?

Comment (mymattcarroll) [reply typescript/forms.d.ts:803]: There is data in S3 that we cannot practically update with `'infoPage'` elements and if we remove the type, we need to have a bunch of `@ts-expect-error` statements in all the places that render historical data.

**Pull request merged**